spec/inject_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../lib/inject'
+
+describe Array do
+
+  context ".injector, when called upon an array" do
+    
+    it "[1,2,3,4,5].injector(0) {+ operation} to eq 15" do
+      expect([1, 2, 3, 4, 5].injector(0){|x, y| x + y}).to eq(15)
+    end
+
+    it "[1,2,3,4,5].injector(5) {+ operation} to eq 20" do
+      expect([1, 2, 3, 4, 5].injector(5){|x, y| x + y}).to eq(20)
+    end
+
+    it "[9,8,7,6,5].injector(0) {- operation} to eq -35" do
+      expect([9,8,7,6,5].injector(0){|x, y| x - y}).to eq(-35)
+    end
+
+    it "[9,8,7,6,5].injector(5) {- operation} to eq -30" do
+      expect([9,8,7,6,5].injector(5){|x, y| x - y}).to eq(-30)
+    end
+
+    it "[9,8,7,6,5].injector(0) {* operation} to eq 0" do
+      expect([9,8,7,6,5].injector(0){|x, y| x * y}).to eq(0)
+    end
+
+    it "[9,8,7,6,5].injector(5) {* operation} to eq 75600" do
+      expect([9,8,7,6,5].injector(5){|x, y| x * y}).to eq(75600)
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).